### ISSUE TITLE
K8S cluster expand/shrink fails after metadata migration on all cloud providers

### ISSUE THREAD

**Issue by vplanche-cssi** [CS-SI/SafeScale#111]
```
operations.(*Feature).Check() [/home/pirate/GoProjects/SafeScale/lib/server/resources/operations/feature.go:392]: failed to check if Feature 'kubernetes' is installed on Cluster 'migra6': ba28e2ac-443f-4563-aabd-8db0ee99500a: failed to execute command on Host 'migra6-node-1' in 00h14m00.000s: failed to execute command on Host 'migra6-node-1' after 00h14m00.000s: reached timeout of 00h14m00.000s: timeout of 14m0s waiting for Task 'ba28e2ac-443f-4563-aabd-8db0ee99500a+/ssh/run' (timeout: 14m0s)
```

**Comment by vplanche-cssi** [comment]
bash_library function 'waitForUserdata' does not handle correctly state of nodes from pre-v21.05.

**Comment by vplanche-cssi** [comment]
Fixed in commit c1bced4cfc3f9b3243db0ce46f29f27dfd1c52d0 in branch develop
